components/Button/index.js

```diff
@@ -0,0 +1,21 @@
+import styled from "styled-components";
+
+export const Button = ({ setProduct, produkt, productToBuy }) => {
```

Comment:
Why are you using the `productToBuy`-prop here? If you are only using it for the console.log you might think of removing both since it is not necessary to let the console.log inside your code.